docs/syscall.md

```diff
@@ -0,0 +1,19 @@
+# System Calls
+
+## Experimental Display and Event System Calls
+
+These system calls are experimental for demonstration of RISC-V graphics applications, they don't exist in the ABI interface of POSIX and Linux, and are only for the convenience of accessing the SDL library.
+
+### `syscall_draw_frame(screen, width, height)`
```

Comment:
The subsection should be `draw_frame`, and you ought to describe the syscall number and usage.